script/deploy

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+set -e
+set -x
+
+USAGE="$0: appname"
+if [ $# -lt 1 ]; then
+  echo "Error: appname is required.\n$USAGE" >&2
+  exit 1
+fi
+
+# http://stackoverflow.com/a/677212/358804
+if ! hash cf-blue-green 2>/dev/null; then
+  npm install -g cf-blue-green
+fi
+
+cf-blue-green $1 -f web-manifest.yml
```

Comment:
AFAIK, `cf-blue-green` doesn't seem to support the `-f` flag. From the docs:

> cf-blue-green creates a temporary manifest from your live application, meaning that it ignores the manifest.yml in your directory, if you have one. To deploy any changes to your manifest, use cf push directly.

For that reason, the docs recommend using a different blue-green helper if possible. https://github.com/concourse/autopilot and https://github.com/bluemixgaragelondon/cf-blue-green-deploy both look decent (we're using autopilot for parts of FEC), and have the advantages of being CF CLI plugins with unit tests.